examples/web_agent/README.md

```diff
@@ -0,0 +1,47 @@
+# WEBAGENT
+
+This is an agent that uses python to connect to Airtop, create sessions, execute prompts on the site and store the data in a local database.
+
+## PREREQUISITES
+
+- PYTHON 3.10
+
+To install, just run the following after downloanding the code
+
+`poetry install`
+
+These are the steps necessary to run the code:
+
+## 1. CREATE LOCAL DATABASE
+
+To create the local database it's necessary to run
+
+`python connect_database.py`
+
+## 2. CREATE WORKER
```

Comment:
the instructions feel strange. Why is it in #2 theres nothing to do, but in #3 there is?